Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -1509,6 +1522,37 @@ namespace O3DE::ProjectManager
         return result && refreshResult;
     }
 
+    AZ::Outcome<QStringList, AZStd::string> PythonBindings::GetIncompatibleProjectGems(
+        const QStringList& gemPaths, const QStringList& gemNames, const QString& projectPath)
+    {
+        QStringList incompatibleGems;
+        bool result = ExecuteWithLock(
+            [&]
+            {
+                using namespace pybind11::literals;
+                auto pythonIncompatibleResult =
+                    m_projectManagerInterface.attr("get_incompatible_project_gems")(
+                        "gem_paths"_a = QStringList_To_Py_List(gemPaths),
+                        "gem_names"_a = QStringList_To_Py_List(gemNames),
+                        "project_path"_a = QString_To_Py_String(projectPath)
+                    );
+
+                // Returns an exit code so boolify it then invert result
+                for (auto& incompatibleGem : pythonIncompatibleResult)
```

Comment:
This should be a const reference.
```suggestion
                for (const auto& incompatibleGem : pythonIncompatibleResult)
```